charts/node-red/templates/deployment.yaml

```diff
@@ -25,6 +25,17 @@ spec:
       serviceAccountName: {{ include "node-red.serviceAccountName" . }}
       securityContext:
         {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      {{- if .Values.persistence.subPath }}
```

Comment:
Can you use here `.Values.persistence.enabled`. As we use this for the whole PVC. creaton.

Reply:
Sorry, i might have been a bit confused when updating the pull request. Fixed now

df1430aa19847e14bb40b13cb04ba9755cb54dfa